src/test/java/com/karankumar/booksapi/repository/BookRepositoryTest.java

```diff
@@ -105,8 +107,47 @@ private Book createBookwithISBN() {
         book.setIsbn13(ISBN);
         return book;
     }
+  @Test
+    @DisplayName("find book by title")
+    void findBookByTitle() {
+        // given
+        createAndSaveAuthors();
+        Book book = createBookWithTitle();
+        bookRepository.save(book);
+
+        // when
+        Book result = bookRepository.findBookByTitle(TITLE);
+
+        // then
+        assertThat(result).isEqualTo(book);
+    }
+
+    @Test
+    @DisplayName("find book by title case insensitive")
+    void findBookByTitleCaseInsensitive() {
+        // given
+        createAndSaveAuthors();
+        Book book = createBookWithTitle();
+        bookRepository.save(book);
 
+        // when
+        Book result = bookRepository.findBookByTitle(TITLE.toLowerCase());
+
+        // then
+        assertThat(result).isEqualTo(book);
+    }
+    private Book createBookWithTitle() {
+        Book book = new Book(
+                TITLE,
+                new Author[]{author1, author2},
+                Language.ENGLISH,
+                ""
+        );
+        return book;
```

Comment:
Instead of creating a variable and then returning, could you please return it immediately?